sdk/guides/setup-a-metagraph/building-metagraph-instances/03-building-data-l1-instances.md

```diff
@@ -1,15 +1,19 @@
 ---
-id: building-metagraph-L1-instances
-title: Building metagraph L1 Instances
+id: building-data-l1-instances
+title: Building data L1 Instances
 ---
 
 
-Let’s create our Metagraph L1 😎, our L1 will need you to set up at least 3 instances. 
+Let’s create our Data L1 😎, our L1 will need you to set up at least 3 instances. 
```

Comment:
Same here. Are we creating all the layers on the same 3 servers or are we assuming the projects will create 3x4 layers = 12 individual servers?

Reply:
Fixed